index.html

```diff
@@ -147,10 +147,15 @@ <h2>Scope</h2>
         </p>
         <ul>
           <li>Addressing errata, ambiguities, and interoperability problems found in previous versions of the Verifiable Credentials Data Model</li>
-          <li>Data models for credentials, presentations, and proofs</li>
-          <li>Registries for the data models</li>
+          <li>A data model for Verifiable Credentials, inclusive of:</li>
+            <ul>
+                <li>credentials and verifiable credentials</li>
+                <li>presentations and verifiable presentations</li>
```

Comment:
it's quite hard to know without the context why there is a differentiation between credentials and verifiable credentials; and between presentations and verifiable presentations. 
I think it would be helpful if we could clarify that we are defining a data model of a credentials, how it can be integrity protected/signed to be verifiable; and data model and integrity protection/signing of proof of possession (presentation and verifiable presentation)?

Reply:
> it's quite hard to know without the context why there is a differentiation between credentials and verifiable credentials; and between presentations and verifiable presentations.

The distinction between a credential and a verifiable credential is made here: https://w3c.github.io/vc-data-model/#credentials

The distinction between a presentation and a verifiable presentation is made here: https://w3c.github.io/vc-data-model/#presentations

> I think it would be helpful if we could clarify that we are defining a data model of a credentials

The current specification seems to do that already and the charter seems to indicate that we're going to continue doing that?

> how it can be integrity protected/signed to be verifiable

That is covered in the current specification here: https://w3c.github.io/vc-data-model/#proofs-signatures

... and the presumption is that we're going to refine that (thus the Data Integrity work item).

> data model and integrity protection/signing of proof of possession (presentation and verifiable presentation)?

I do think that's in scope as well.

The reason we list all of these things out above is because @David-Chadwick insisted that we be precise about what is included in the data model, which was a good idea.

Perhaps I'm missing your point @Sakurann... could you provide concrete text that we might use in addition to/in place of the text above?